### ISSUE TITLE
TypeError: bug1258464killer() got multiple values for keyword argument 'default_file'

### ISSUE THREAD

**Issue by akuzminsky** [twindb/proxysql-tools#29]
* ProxySQL Tools version: 0.3.4
* Python version: 2.7.5
* Operating System: CentOS 7

### Description

proxysql-tool galera bug1258464killer crashes.

### What I Did

```
# proxysql-tool galera bug1258464killer
Traceback (most recent call last):
  File "/bin/proxysql-tool", line 11, in <module>
    sys.exit(main())
  File "/usr/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args[1:], **kwargs)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
TypeError: bug1258464killer() got multiple values for keyword argument 'default_file'
```

**Comment by mkryva** [comment]
May You try to check it on feature/issue-29 with args in next sequence:
proxysql_tools galera bug1258464killer —default-file <path_to_default_file>

**Comment by akuzminsky** [comment]
Fixed in 0.3.10
